swarm/network/hive.go

```diff
@@ -123,7 +124,13 @@ func (h *Hive) Stop() error {
 	log.Info(fmt.Sprintf("%08x hive stopping, saving peers", h.BaseAddr()[:4]))
 	h.ticker.Stop()
 	if h.Store != nil {
-		return h.savePeers()
+		if err := h.savePeers(); err != nil {
+			return fmt.Errorf("could not save peers to persistence store: %v", err)
+		}
+		if err := h.Store.Close(); err != nil {
+			log.Error(fmt.Sprintf("could not close file handle to persistence store: %v", err))
```

Comment:
i thought i commented on this already, do not log errors you return

Reply:
came back after rebasing